### DISCUSSION TITLE
Wiki links with relative paths resolve differently in preview and saved page

### DISCUSSION THREAD

**Discussion by mmannion** [community/community#5570] [Projects and Issues]
I'm sipping from the GitHub firehose at the moment, so I hope this is the appropriate place to post this question. If not, I'd appreciate a reference to the appropriate forum.

# Use case

I need to create links to files in a GitHub repo from pages in the repo's wiki.

I understand that there are several constructs for building a valid URL that references a file in the repo. From what I've read in other discussions, using relative paths is advisable for portability / future-proofing.

Specifically, the style of relative path I'm using is `![Icon](../raw/main/icon.png)`

Note that this relative path navigates only one level up, `../`, from the wiki page's location.

# Issue

GitHub resolves the relative path to different absolute URLs when:
* Viewing the saved wiki page `https://github.com/user/repo/raw/main/icon.png`
* Previewing the page during editing `https://github.com/user/repo/wiki/main/icon.png`

Note the presence of the extra `wiki` directory in the path in the latter case. This results in a broken link when viewing the saved page.

Conversely, navigating two levels up, `../../`, causes the link to work properly when previewing, but improperly when viewing the saved page.

# Questions

* Is there a syntax that 1) allows using a relative pathname, and 2) resolves properly both when previewing the page during editing, and when viewing the saved page?

* If not, what is the appropriate way for me to log this as a bug / enhancement request?

Thanks!

**Comment by TWolversonReply** [comment]
I've encountered the same issue today, in this case creating the organization private profile and linking to other .md files from readme.md. The profile when rendered expects relative links to contain /profile/ but this doesn't work when previewing because that isn't actually the correct relative path between files that are under the same directory.

**Comment by Kaloszer** [comment]
Having the same issue, preview shows valid if you navigate to [] (../.media/someimg.png) but does not show in actual wiki

**Comment by ossy-szeged** [comment]
Preview links are still wrong. :(

**Comment by regorxxx** [comment]
Having the same problem, links don't work properly.
